src/server/plugins/engine/components/ComponentCollection.ts

```diff
@@ -250,14 +246,6 @@ export class ComponentCollection {
       return { type, isFormComponent, model }
     })
 
-    if (conditions) {
-      return result.filter((item) =>
-        item.model.condition
-          ? conditions[item.model.condition]?.fn(payload)
-          : true
-      )
-    }
-
```

Comment:
As shown above, this is dead code

We never passed in the 3rd `conditions` param anywhere